pom.xml

```diff
@@ -68,6 +68,7 @@
       <module>artemis-features</module>
       <module>artemis-quorum-api</module>
       <module>artemis-quorum-ri</module>
+      <module>artemis-image</module>
```

Comment:
Probably wouldnt want this to be building on every build, so maybe shift to release profile (needs to be enabled then to update the versions)